CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ option(ARMA_EXTRA_DEBUG "Compile with extra Armadillo debugging symbols." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
 option(BUILD_TESTS "Build tests." ON)
 option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
-option(DISABLE_DOWNLOADS "Disable downloads of dependencies during build." OFF)
+option(DISABLE_DOWNLOADS "Disable downloads of dependencies during build." ON)
```

Comment:
To fix the Jenkins build, we'll at least need to make sure that `STB_IMAGE_INCLUDE_DIR` is only appended to `MLPACK_INCLUDE_DIRS` in the block `if (STB_IMAGE_FOUND)`.